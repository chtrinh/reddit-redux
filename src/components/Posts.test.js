import React from 'react';
import Posts from './Posts';

it('renders without crashing', () => {
  const posts = [
    { url: 'http://example.com/blah', title: 'blah' },
    { url: 'http://example.com/others', title: 'others' }
  ];

  const wrapper = shallow(<Posts posts={posts} />);

  expect(wrapper).toMatchSnapshot();
});
