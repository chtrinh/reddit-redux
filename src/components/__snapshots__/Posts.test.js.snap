// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<ul>
  <li>
    1
     - 
    <a
      href="http://example.com/blah"
    >
      blah
    </a>
  </li>
  <li>
    2
     - 
    <a
      href="http://example.com/others"
    >
      others
    </a>
  </li>
</ul>
`;
